lib/Stats.js

```diff
@@ -630,20 +627,12 @@ class Stats {
 				colors.normal(prefix);
 				let sum = 0;
 				const path = [];
-				let current = module;
-				while(current.issuer) {
-					path.unshift(current = current.issuer);
+				if(module.issuer) {
```

Comment:
@sokra, do you mean move the deleted code to [here](https://github.com/webpack/webpack/blob/master/lib/Stats.js#L177)